client/src/components/ManageAccount/ManageAccount.module.css

```diff
@@ -44,7 +44,7 @@
     width: var(--column-width--small);
 
     /* To ensure these 3 panels are always in the same column together */
-    max-height: calc(33% - var(--vertical-buffer));
+    max-height: calc(50% - var(--vertical-buffer));
```

Comment:
**Problem.** This code change makes the comment inaccurate. 

<details>
<summary>How is Comment Inaccurate?</summary>

The "3 panels" are "Licenses", "3rd Party Apps", and "Password" ([reference](https://xd.adobe.com/view/db2660cc-1011-4f26-5d31-019ce87c1fe8-ad17/screen/62ca01fd-4444-4d1c-aeb8-6db5aac20590/)). With this change, the 3 panels can exceed 33% height, thus overflow one column.

The result is not easily seen, because the Portal may not show as many panels and can exist in this section ([design](https://xd.adobe.com/view/db2660cc-1011-4f26-5d31-019ce87c1fe8-ad17/screen/62ca01fd-4444-4d1c-aeb8-6db5aac20590/)).

</details>

<details>
<summary>Proposed Solutions</summary>

- (a) update comment to be accurate
- (b) explain why comment is temporarily inaccurate

    <details>
    <summary>Examples</summary>

    - ```suggestion
          /* CAVEAT: your_explanation_here */
          max-height: calc(50% - var(--vertical-buffer));
        ```
    - ```suggestion
          /* CAVEAT: 50% instead of 30% allows different layout until FP-1329 */
          max-height: calc(50% - var(--vertical-buffer));
        ```

    </details>

</details>

Reply:
Yes, this is what I understood, and I couldn't get this to work with the error message rendered along with the rest of these tables. In the example below, I was attempting to push the 'Optional Information' table into the next column, and I found that changing this percentage allowed that without any changes to the page without the error message showing.

I do understand the point, and see that this ratio does not work when 'Licenses' and/or 'Third party apps' are large, I am just unsure what to include here which could conditionally alter the layout should an error pop up.

Reply:
<details>
<summary>All is well.</summary>

Yup, I understand. No need to solve it now—it was an unexpectedly complicated layout problem—I created a ticket [FP-1329](https://jira.tacc.utexas.edu/browse/FP-1329) for later. I'll take this burden.

</details>

<details>
<summary>I was just hoping for an update related to the comment.</summary>

My "Proposed Solutions" are just to (a) update comment to be accurate or (b) add comment to "explain why comment is temporarily inaccurate". If you're not sure what to put, feel free to use either of the Examples in the "Proposed Solutions".

</details>

Alternatively, retain the comment inaccuracy and I'll re-evaluate it in [FP-1329](https://jira.tacc.utexas.edu/browse/FP-1329).

Reply:
Also, see [Alex's suggestion](https://github.com/TACC/Core-Portal/pull/543#pullrequestreview-803221938).